aws/v13.1.0/release.yaml

```
@@ -0,0 +1,69 @@
+# Generated with:
+# devctl release create --name 13.1.0 --provider aws --base 13.0.0 --component aws-operator@9.3.6 --overwrite
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v13.1.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.5.0
+  - componentVersion: 1.0.2
+    name: cert-manager
+    version: 2.4.1
+  - name: chart-operator
+    version: 2.6.0
+  - componentVersion: 1.18.1
+    name: cluster-autoscaler
+    version: 1.18.3
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.2.0
+  - componentVersion: 0.7.6
+    name: external-dns
+    version: 1.6.0
+  - componentVersion: 3.6.0
+    name: kiam
+    version: 1.7.0
+  - componentVersion: 1.9.7
+    name: kube-state-metrics
+    version: 1.3.0
+  - componentVersion: 0.4.1
+    name: metrics-server
+    version: 1.2.1
+  - name: net-exporter
+    version: 1.9.2
+  - componentVersion: 1.0.1
+    name: node-exporter
+    version: 1.7.1
+  components:
+  - name: app-operator
+    releaseOperatorDeploy: true
+    version: 3.1.0
```

```suggestion
    version: 3.2.0
```

@paurosello There is a problem with finalizers in this release that @xh3b4sd hit testing on ginger.

In 3.1.0 the finalizer got changed from `operatorkit.giantswarm.io/app-operator` to `operatorkit.giantswarm.io/app` which is wrong and causes problems on deletion.

In 3.2.0 we use the correct value which is `operatorkit.giantswarm.io/app-operator-app` and there is migration logic to use the new value.

I'll do a PR for requests but posting here so we don't hit this.





The finalizer got renamed from